docs/installation.md

```diff
@@ -18,7 +18,7 @@ Next, create the environment and start it up:
     cd cla_public
     virtualenv env --prompt=\(cla_public\)
 
-    source env/bin/activate
```

Comment:
Why are you removing this? It's handy to have.

Reply:
It shouldn't be removing that, will amend